lms/djangoapps/ora_staff_grader/mock/utils.py

```diff
@@ -41,36 +41,41 @@ def get_updated_dict(dict_to_update, key_list, value):
 
 
 def get_course_metadata(ora_location):
-    """ Get course metadata, indexed by ORA block location """
+    """Get course metadata, indexed by ORA block location"""
     return read_data_file("course_metadata.json")[ora_location]
 
 
 def get_ora_metadata(ora_location):
-    """ Get ORA metadata, indexed by ORA block location """
+    """Get ORA metadata, indexed by ORA block location"""
     return read_data_file("ora_metadata.json")[ora_location]
 
 
 def get_submissions(ora_location):  # pylint: disable=unused-argument
-    """ Get Submission list, scoped by ORA block location """
+    """Get Submission list, scoped by ORA block location"""
     submissions = read_data_file("submissions.json")[ora_location]
 
     # For the list view, we don't return grade data
-    for submission_id, submission in submissions.items():  # pylint: disable=unused-variable
-        submission.pop('gradeData')
+    for (
```

Comment:
What's the issue here? Line too long? You can move the pylint disable to the line above the for to avoid this kinda gross linebreak. or just use .values() ranther than .items()